servicetalk-utils-internal/src/main/java/io/servicetalk/utils/internal/PlatformDependent.java

```diff
@@ -277,12 +278,13 @@ private static final class Queues {
         static {
             // Internal property, subject to removal after validation completed.
             final String useUnpaddedQueuesName = "io.servicetalk.internal.queues.useUnpadded";
-            boolean useUnpaddedQueues = getBoolean(useUnpaddedQueuesName);
+            boolean useUnpaddedQueues = parseBoolean(getProperty(useUnpaddedQueuesName, "true"));
```

Comment:
To minimize changes, wdyt if we keep this flag `false` for the next release and change it to `true` in release after?

Reply:
the flag has been false since the feature was introduced. The behavior should be the same in the `ea` and regular release, and the intention was to move default behavior to conserve memory (bcz our queues are typically small so constant queue overhead is relatively large). we can defer flipping the default if you prefer but I don't think going between `ea` and regular changes any behavior.

Reply:
The behavior will be the same only for those who already tried to set the flag to `true` and had exactly 3.3.1-ea in the classpath. In my experience, some users don't like `ea` in their classpath and either downgraded the version to 3.3.0 or already upgraded to 4.0.1.
The safer approach will be to update the version in one release and start using the new feature by default in another release.

Reply:
Follow up: #2490